pkg/cloud_provider/user_tags.go

```diff
@@ -0,0 +1,523 @@
+/*
+Copyright 2018 The Kubernetes Authors.
```

Comment:
Please update the year in the author comment

Reply:
suggestion incorporated.